sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -7632,6 +7633,25 @@ def test_Str():
     from sympy.core.symbol import Str
     assert pretty(Str('x')) == 'x'
 
+
+def test_transforms():
+    e = LaplaceTransform(a, b, c)
+    assert pretty(e) == \
+    'L [a](c)\n'\
+    ' b      '
+    assert upretty(e) == \
+    'ℒ [a](c)\n'\
+    ' b      '
+
+    e = SineTransform(a, b, c)
+    assert pretty(e) == \
+    'SIN [a](c)\n'\
+    '   b      '
+    assert upretty(e) == \
+    '𝒮ℐ𝒩 [a](c)\n'\
+    '   b      '
```

Comment:
I'm not familiar with the sin transform. Is this common notation?

Reply:
It is the same as used in the LaTeX-printer. But it seems like Wikipedia prefers other notation (although it is not really clear how to get something similar with both function and variable with this). https://en.wikipedia.org/wiki/Sine_and_cosine_transforms

Reply:
This is not a blocker from me so +1 to merge if you're done with this. I was just asking because I haven't seen this before.

Reply:
Will just sort out the inverse transform when I have time to look into how that is done in the pretty printer...